cli/command/system/prune.go

```diff
@@ -92,19 +117,19 @@ func runPrune(dockerCli command.Cli, options pruneOptions) error {
 		pruneFuncs = append(pruneFuncs, runBuildCachePrune)
 	}
 
-	var spaceReclaimed uint64
```

Comment:
nit: I would rather keep the variable declaration immediately before the first use, instead of at the start of the function. Declaring everything at the top of a function makes the scope less obvious and makes it harder to refactor later.

Reply:
Corrected.